backend/src/test/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapperTest.java

```diff
@@ -109,6 +105,7 @@ void map_WhenSingleMaterialItemStock_ReturnsItemStockSamm() {
         // Given
         Material semiconductorMaterial = Material.builder()
             .ownMaterialNumber(CUSTOMER_MAT_NUMBER)
+            .materialNumberCx(CX_MAT_NUMBER)
```

Comment:
Why did you insert a materialNumberCx here? We are setting up a MaterialItemStock from a Customer's perspective. The only relevant CX-Id here should be the Supplier's CX-Number.